app/src/main/java/org/jellyfin/mobile/player/PlayerViewModel.kt

```diff
@@ -68,6 +70,14 @@ import org.koin.core.qualifier.named
 import timber.log.Timber
 import java.util.concurrent.atomic.AtomicBoolean
 
+/**
+ * Represents the type of decoder
+ */
+enum class DecoderType {
+    HARDWARE,
+    SOFTWARE,
+}
```

Comment:
Should be moved to another file.

Reply:
Thanks, I think it would be better to have it in the `ui` sub-package, though. The `interaction` package is meant for connecting outside components like the web-interface to the player.